packages/@n8n/config/src/configs/runners.config.ts

```diff
@@ -53,4 +53,12 @@ export class TaskRunnersConfig {
 	/** Should the output of deduplication be asserted for correctness */
 	@Env('N8N_RUNNERS_ASSERT_DEDUPLICATION_OUTPUT')
 	assertDeduplicationOutput: boolean = false;
+
+	/** Max duration (in seconds) a task is allowed to take before being aborted. */
+	@Env('N8N_RUNNERS_TASK_TIMEOUT')
+	taskTimeout: number = 60;
+
+	/** How often (in seconds) the runner is expected to send a heartbeat to the broker. */
```

Comment:
I find this comment to be a bit backwards. Would this make more sense?

```suggestion
	/** How often (in seconds) to send a heartbeat to runners to check that they are still healthy. */
```

Reply:
I phrased it like this because this config is from main's perspective and the heartbeat sender is the runner. The heartbeat is not the main's ping but the runner's pong right?